web/index.jsp

```diff
@@ -29,6 +29,10 @@ include file="projects.jspf"
 {
     PageConfig cfg = PageConfig.get(request);
     cfg.setTitle("Search");
+    String sourceRootPath = cfg.getSourceRootPath();
```

Comment:
I thought checking sourceRootPath value should be enough, since there is only one way to make 100% sure to check if file is readable is to read it in try catch block ( http://stackoverflow.com/questions/11901382/how-to-check-if-a-file-is-readable) which seems like overkill